db/migrations/postgres/000105_remove_tokens.up.sql

```diff
@@ -0,0 +1,13 @@
+DO $$
+BEGIN
+WITH oauthDelete AS (
+	DELETE FROM oauthaccessdata o
+	WHERE NOT EXISTS (
+		SELECT p.* FROM preferences p
+		WHERE o.clientid = p.name AND o.userid = p.userid AND p.category = 'oauth_app' 
+		and p.name IS NULL
```

Comment:
(question) Since the preference was deleted in the original code, is the `and p.name IS NULL` part correct in the subquery? We intend to delete oauthaccessdata that don't have a relevant preference (ie they have gone through  `DeauthorizeOAuthAppForUser` before right?)